gender_analysis/gender.py

```diff
@@ -136,3 +142,66 @@ def get_pronouns(self):
                 pronouns.add(pronoun)
 
         return pronouns
+
+    def uses_name(self, name):
+        """
+        Performs a check for whether the name is associated with the gender.
+
+        Note that names are case-sensitive.
+
+        >>> from gender_analysis.pronouns import PronounSeries
+        >>> from gender_analysis.gender import Gender
+        >>> he_series = PronounSeries('Masc', {'he', 'him', 'his'})
+        >>> masc_names = ['Andrew', 'Richard', 'Robert']
+        >>> male = Gender('Male', he_series, names=masc_names)
+        >>> male.uses_name('Richard')
+        True
+        >>> male.uses_name('robert')
+        False
+
+        :param name: A string name to search for
+        :return: A boolean indicating whether the name is associated with the gender
+        """
+
+        return name in self.names
+
+    def uses(self, identifier):
```

Comment:
Feel similarly about this as `uses_name` -- this is just an `in` on `self.associations`. (And actually, if we leave it in, it should be implemented like that, no? Otherwise we have redundant implementations of the `associations` concept.)